ethcore/src/state/substate.rs

```diff
@@ -28,9 +28,6 @@ pub struct Substate {
 	/// Any accounts that have suicided.
 	pub suicides: HashSet<Address>,
 
-	/// Any accounts that are tagged for garbage collection.
-	pub garbage: HashSet<Address>,
```

Comment:
i'm a bit concerned that the alteration of semantics (from explicit collection of gardbage-collected accounts to the implicit reliance on touching the cache) here will lead to subtle consensus flaws. i don't remember exactly, but when i coded this originally i chose this means of doing it for a reason.